pint_server/database.py

```diff
@@ -154,7 +154,7 @@ def create_postgres_url_from_env():
 
 
 def init_db(dbconfig=None, outputfile=None, echo=None,
-            hide_parameters=None):
+            hide_parameters=None, create_all=True):
```

Comment:
As per my comments above, perhaps this should default to false rather than true? The production workflow should only ever be starting the application side of things against an already initialised DB, and shouldn't need to create any tables...

Reply:
The only reason I default it to `True` was to preserve backward compatibility. If there's no such concern then we should default it to False.

Reply:
Default to False.